src/main/features/claim/routes/claimant-add.ts

```diff
@@ -0,0 +1,59 @@
+import * as express from 'express'
+import { Paths } from 'claim/paths'
+
+import { Form, FormValidationError } from 'app/forms/form'
+import { FormValidator } from 'app/forms/validation/formValidator'
+import ErrorHandling from 'common/errorHandling'
+import { YesNo } from 'app/forms/models/yesNo'
+
+import { ClaimDraftMiddleware } from 'claim/draft/claimDraftMiddleware'
+import { ValidationError } from 'class-validator'
+import { ClaimantAddition } from 'app/forms/models/claimantAddition'
+import { Claimants } from 'common/router/claimants'
+
+const MAX_CLAIMANTS_ALLOWED: number = 20
+const ERROR_MESSAGE: string = `You can't add more than ${MAX_CLAIMANTS_ALLOWED} claimants`
+
+function renderView (form: Form<ClaimantAddition>, res: express.Response) {
+  const claimants = res.locals.user.legalClaimDraft.claimants
+
+  res.render(Paths.claimantAdditionPage.associatedView, {
+    form: form,
+    claimants: res.locals.user.viewDraft.isClaimantDeleted || claimants.length > 1 ? claimants : null,
```

Comment:
what will happen if we have `claimants.length == 1` ?

Reply:
claimants table is not displayed

Reply:
that's fine now. previously check was > 2.

Reply:
ok. fine now.